### ISSUE TITLE
When the window loses focus io.KeysDown should be set to false

### ISSUE THREAD

**Issue by 1025798851** [ocornut/imgui#3961]
Version: 1.83 WIP
Branch: 1.83 WIP_docking_

Back-ends: imgui_impl_win32.cpp
Operating System: Windows10 20H2

```cpp
IMGUI_IMPL_API LRESULT ImGui_ImplWin32_WndProcHandler(HWND hwnd, UINT msg, WPARAM wParam, LPARAM lParam)
{
...
    case WM_KILLFOCUS:
        memset(io.KeysDown, 0, sizeof(io.KeysDown));
        return 0;
...
}
```

**Comment by ocornut** [comment]
Same as #2062

**Comment by ocornut** [comment]
I'll move to #2062

**Comment by ocornut** [comment]
I pushed your suggested fix now : 1491d2c9168a1982ea88dd638509e64ec990daf7

As it happens, in docking + multi-viewports enabled it does make held key flick off for one frame, but this issue has also been present in the GLFW backend so my conclusion is that it's a much less problematic issue than the one we are fixing one. We can improve on the multi-viewports code later.
